paths/job_locales/create.yaml

```diff
@@ -73,4 +73,18 @@ requestBody:
               type: string
             example:
             - abcd1234cdef1234abcd1234cdef1234
+          team_ids:
```

Comment:
This should be called `translator_team_ids`

Reply:
In the ticket on TP, it specified the parameter should be called `team_ids`

Reply:
I believe it will be teams for the `#show` where teams combine translators and reviewers

Reply:
This ticket has the good specification, where do you see it? @orwigeg 

![Screenshot from 2023-02-16 09-11-50](https://user-images.githubusercontent.com/1395527/219305924-f0d0c3f7-c8a5-408d-9b64-04dc53067ab0.png)